Amiga quick start guide (#87)
farm-ng/amiga-dev-kit: website/docs/debug_cable/README.md

# farm-ng Debug Cable

![IMG_9574](https://user-images.githubusercontent.com/64480560/205400778-1bfa084c-a187-4f42-8840-7b91714ab7f0.jpg)
## Overview

Cable used to update and debug farm-ng products. It includes rest functionality along with USB and serial communication.

The larger circuit board on the left is the USB / debug breakout, and the smaller board on the right is the serial interface.

![IMG_9578](https://user-images.githubusercontent.com/64480560/205400599-f79e0cae-35f3-4610-bbcf-9a4e50857fac.jpg)

### Product specific uses

#### How to Connect
![187535995-d4ad8c7c-05a0-482b-8be9-6b6fb6b87ac3](https://user-images.githubusercontent.com/64480560/206007745-b36c59c7-22dd-4435-9cae-8503956174f3.png)

**Connect cable to the debug port on the back of dashboard**

> #### Dash

Reviewer: ```suggestion
#### Dashboard
```

- USB interface is for REPL, filesystem access as well as updating firmware and applications,
- Serial interface is not used at the moment
- Reset is connected to the Microcontroller reset and used for hard reset. It also can be double pressed to force bootloader mode.

> **See update process documentation for more information [here](https://amiga.farm-ng.com/docs/dashboard/dashboard-fw)**

Reviewer: Should be relative link: https://docusaurus.io/docs/markdown-features/links

```suggestion
**See update process documentation for more information [here](/docs/dashboard/fw_updates.md)**
```

#### Brain

- USB interface is used for connecting for lashing images
- Serial interface is used for operating system terminal
- Reset / Flash button is used for enabling the firmware update mode (press on boot)

> **See flashing and operating system documentation for more information [here](https://amiga.farm-ng.com/docs/dashboard/dashboard-fw)**

Reviewer: File path link. Also not sure what you mean here...
```suggestion
**See flashing and operating system documentation for more information [here](/docs/brain/README.md)**
```
